lib/ethpy/ethpy/hyperdrive/api_test.py

```diff
@@ -1,78 +1,79 @@
 """Tests for hyperdrive/api.py"""
+# fixtures require redefining the import name and unused import
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-import
 from __future__ import annotations
 
 from typing import cast
 
 from eth_typing import URI
 from ethpy import EthConfig
 from ethpy.base.transactions import smart_contract_read
-from ethpy.hyperdrive.addresses import HyperdriveAddresses
-from ethpy.hyperdrive.api import HyperdriveInterface
-from ethpy.test_fixtures.local_chain import LocalHyperdriveChain
+from ethpy.hyperdrive import DeployedHyperdrivePool, HyperdriveAddresses, HyperdriveInterface
+from ethpy.test_fixtures.local_chain import local_hyperdrive_pool
```

Comment:
Test fixtures should be added to `conftest.py` instead of imported. This should allow for removing the redefined-outer-name and unused-import above